### PR TITLE
[add] check Duplicate

### PR DESCRIPTION
**Pull request by kazukazuinaina**
Hello.
This patch also check to duplicate.
Could you check this Pull Request?

### CODE REVIEW

**Comment by kazukazuinaina** [comment]
If you don't need this pull request, feel free to close it. 🙇

File: plugin/nvim_lsp.vim [hunk 1]
```diff
@@ -1,3 +1,8 @@
+if exists('g:nvim_lsp')
+  finish
+endif
+let g;nvim_lsp = 1
```
Comment (lithammer) [inline plugin/nvim_lsp.vim]:
```suggestion
let g:nvim_lsp = 1
```

Comment (kazukazuinaina) [reply plugin/nvim_lsp.vim]: Oh! sorry, I mistaked.
I fixed.

**Comment by kazukazuinaina** [comment]
Should I need squash?

**Comment by h-michael** [comment]
> Should I need squash?
@kazukazuinaina Please squash if possible.

**Comment by kazukazuinaina** [comment]
@h-michael 
Ok, I sqhashed!

**Comment by kazukazuinaina** [comment]
@h-michael 
Thanks for review

**Pull request merged**